framework/math/quadratures/quadrature.cc

```diff
@@ -10,7 +10,7 @@ Quadrature::GetInputParameters()
 {
   InputParameters params = Object::GetInputParameters();
 
-  params.SetGeneralDescription("\\defgroup chi_math__Quadrature\n"
+  params.SetGeneralDescription("\\defgroup math__Quadrature\n"
```

Comment:
does the dunder (__) have a meaning?

Reply:
Maybe, it is a thing for doxygen. I believe that this can be removed since we will be using sphinx.

Reply:
I talked with Jan about this a long time ago. It has to do with doxygen and/or some scripts he had to generate documentation.